src/openMVG/sfm/pipelines/global/sfm_global_engine_relative_motions.cpp

```diff
@@ -484,7 +484,7 @@ void GlobalSfMReconstructionEngine_RelativeMotions::Compute_Relative_Rotations
   #pragma omp parallel for schedule(dynamic)
 #endif
   // Compute the relative pose from pairwise point matches:
-  for (int i = 0; i < poseWiseMatches.size(); ++i)
+  for (size_t i = 0; i < poseWiseMatches.size(); ++i)
```

Comment:
ok sorry it must have slipped in when i did the rebase and solved some conflicts